core/services/players/models.py

```diff
@@ -16,7 +16,7 @@ class RegionEnum(int, enum.Enum):
 
 class Player(SQLModel):
     __table_args__ = dict(mysql_charset="utf8mb4", mysql_collate="utf8mb4_general_ci")
-    id: int = Field(primary_key=True)
+    id: Optional[int] = Field(primary_key=True, sa_column=Column(Integer, primary_key=True, autoincrement=True))
```

Comment:
```suggestion
    id: Optional[int] = Field(None, primary_key=True, sa_column=Column(Integer, primary_key=True, autoincrement=True))
```

这里最好给 `id` 写个 `None` ，防止 IDE 代码提示报错